websites/T/Tanks.gg/presence.ts

```diff
@@ -0,0 +1,117 @@
+const presence = new Presence({
+		clientId: "1003195428642238464",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/jjvJi16.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href } = window.location;
+
+	if (pathname === "/") presenceData.details = "Browsing the home page";
+	else if (document.location.pathname.startsWith("/tank/")) {
+		presenceData.details = "Viewing a tank";
+		presenceData.state = `${
+			document.querySelector(".header-tank > h1").childNodes[0].textContent
```

Comment:
```suggestion
			document.querySelector(".header-tank > h1").firstElementChild.textContent
```

Reply:
> 

Nooo !!!!
Use `firstChild` instead ! `firstElementChild` will target the first node nested in the current node targeted, without the texte !
So, for example :
```...<h1>Name <small>second name</small> <span>role</span></h1>```
With `document.querySelector(".header-tank > h1").firstElementChild.textContent` the part targeted will be the `small` element, since with `firstChild`, this will be the text "Name".

So use `firstChild`, DarkVillager is right in the fact of using a specific attributes to get it instead of `childNodes[0]`, I was not aware of `firstChild` when I suggested you the change.

Reply:
Thanks for the help, guys